sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+``` 
+
+```shell
+yarn add @fluentui/web-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+  fluentButton, 
+  provideFluentDesignSystem 
+} from "@fluentui/web-components";
+
+provideFluentDesignSystem()
+  .register(
+    fluentButton() // custom element
+  );
+```
+
+---
+
+If you are attempting to configure the components for integration into a specific Microsoft product, the following table provides `AccentBaseColor` values you can use:
+
+| Product     | AccentBaseColor | 
+| :---        | :---            | 
+| Office      | #D83B01         | 
+| Word        | #185ABD         | 
+| Excel       | #107C41         | 
+| PowerPoint  | #C43E1C         | 
+| Teams       | #6264A7         | 
+| OneNote     | #7719AA         | 
+| SharePoint  | #03787C         | 
+| Stream      | #BC1948         | 
+
+---
+
+## [Using Components](https://www.fast.design/docs/components/getting-started)
+
+### Setup
+
+To register design system components:
+
+```ts
+import { 
+  fastButton, 
+  provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+  .register(
+    fastButton() // desired custom component
+  );
+```
+
+Or, register all system components:
+
+```ts
+import { 
+  allComponents, 
+  provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem().register(allComponents);
+```
+
+With the components registered, add any component to the HTML:
+
+```html
+<fast-button>Hello world</fast-button>
+```
+
+---
+
+Launch our [Component Explorer](https://explore.fast.design/) for a comprehensive list of FAST Components.
+
+---
+### [Integrations](https://www.fast.design/docs/integrations/introduction)
+
+FAST libraries can also be used in combination with a wide variety of existing technologies.
+
+- [Angular](https://www.fast.design/docs/integrations/angular)
+- [ASP.NET](https://www.fast.design/docs/integrations/aspnet)
+- [Aurelia](https://www.fast.design/docs/integrations/aurelia)
+- [Blazor](https://www.fast.design/docs/integrations/blazor)
+- [Ember](https://www.fast.design/docs/integrations/ember)
+- [React](https://www.fast.design/docs/integrations/react)
+- [Vue](https://www.fast.design/docs/integrations/vue)
+- [Webpack](https://www.fast.design/docs/integrations/webpack)
+
+Not seeing an integration for your preferred technology?  Open an issue on [GitHub](https://github.com/microsoft/fast/issues).
+
+---
+
+## [Building Components](https://www.fast.design/docs/fast-element/getting-started)
+
+### Setup
+
+To define a custom element:
+
+```ts
+import { FASTElement, customElement } from "@microsoft/fast-element";
+
+@customElement("name-tag") // custom element being created
+export class NameTag extends FASTElement {
+
+}
+```
+
+With this in place, you can now use your `<name-tag>` element anywhere in HTML with the following markup:
+
+```html
+<name-tag></name-tag>
+```
+
```

Comment:
@eisenbergeffect @huynhicode this isn't exhaustive as we have this AND the registration portion, which depending on use case/scenario, either seems reasonable. If I were doing a quick run through this, I would _assume_ this is "the way". How do we capture both in the context of this cheat sheet? Would perhaps linking to something be more effective?

Reply:
I think we could have a short explanation here that says there are two main approaches. The first approach is for simple declarations of non-shared components. The second style is for components designed to be published in shareable libraries. We could link down in the doc to the second approach if it makes sense to put that one somewhere else.

Reply:
@chrisdholt @EisenbergEffect Thank you - I updated the "Building Components" section with the additional verbiage and a link to the Design Systems section.